src/services/item/plugins/importExport/service.ts

```diff
@@ -312,7 +317,12 @@ export class ImportExportService {
             h5pFileStream,
           );
 
-          extra = h5pFileInfo;
+          extra = { [ItemType.H5P]: h5pFileInfo };
+        }
+
+        // Handle the APP item extra
+        if (item.type === ItemType.APP) {
+          extra = { [ItemType.APP]: item.extra[ItemType.APP] };
```

Comment:
```suggestion
          extra = item.extra;
```
?? Or am I missing something?